### ISSUE TITLE
[Batch] pool list command is failing for table output

### ISSUE THREAD

**Issue by troydai** [Azure/azure-cli#4378]
```
09:55 $ az batch pool list
Table output unavailable. Use the --query option to specify an appropriate query. Use --debug for more info.
```

Root cause https://github.com/Azure/azure-cli/blob/56a9a39e513247816a368d0deb8441e436f961e4/src/command_modules/azure-cli-batch/azure/cli/command_modules/batch/_format.py#L208. The property `currentDedicated` doesn't exist.

Here's the potential fix:
```
(env) ✔ ~/code/azure-cli [paging ⭐|❗3…2]
09:55 $ az batch pool list -ojson | grep current
    "currentDedicatedNodes": 2,
    "currentLowPriorityNodes": 0,
    "currentDedicatedNodes": 0,
    "currentLowPriorityNodes": 0,
```

Must be one of those.

**Comment by annatisch** [comment]
@xingwu1 - looks like this is still an issue the Batch CLI - could you take a look?

**Comment by troydai** [comment]
Fixed in #6393
